pre_cache_action.sh

```diff
@@ -44,9 +44,19 @@ fi
 
 # Is length of string zero?
 if test -z "${packages}"; then
-  log "aborted"
-  log "Packages argument cannot be empty." >&2
-  exit 3
+  if [ "$IF_EMPTY_PACKAGES" = "ignore" ]; then
+    exit 0
+  elif [ "$IF_EMPTY_PACKAGES" = "warn" ]; then
+    echo "::warning::Packages argument is empty."
+    exit 0
+  else
+    if [ "$IF_EMPTY_PACKAGES" != "error" ]; then
```

Comment:
IMO we should just let the action catch this. Thoughts?

Reply:
There is no built-in validation for `inputs` parameters (except for `required`), is that correct? But it is fine to regard inputs that are not equal to either `warn` or `ignore` as `error` (I have changed it so).